modules/apps/frontend-js/frontend-js-web/test/liferay/delegate/delegate.es.js

```diff
@@ -181,4 +181,61 @@ describe('delegate', () => {
 
 		expect(delegateTarget).toBe(document.querySelector('.match'));
 	});
+
+	it('works with multiple selectors', () => {
+		document.body.innerHTML = `<div class="match" data-testid="match">
+				<div>
+					<div class="one" data-testid="one"></div>
+					<div class="two" data-testid="two"></div>
+				</div>
+			</div >`;
+
+		const listener = jest.fn();
+
+		const selector = '.one,.two';
+
+		delegate(document.body, 'mouseover', selector, listener);
+
+		const event = new Event('mouseover', {
+			bubbles: true,
+			cancelable: true,
+		});
+
+		let element = getByTestId(document, 'one');
+		element.dispatchEvent(event);
+
+		expect(listener).toHaveBeenCalledTimes(1);
+
+		element = getByTestId(document, 'two');
+		element.dispatchEvent(event);
+
+		expect(listener).toHaveBeenCalledTimes(2);
+	});
+
+	it('delegateTarget returns the correct selector', () => {
```

Comment:
Could this be made more specific too?

We already have `it('gives access to delegateTarget inside the callback'` higher up. What's the specific thing you're trying to check here? (Also, nit: note that it's not `return`-ing anything, and it's an element, not a selector.)

Ideally, whatever you come up with should read like a valid English sentence when read out loud (unlike "it delegateTarget returns"). Maybe something like:

```js
it('sets delegateTarget correctly when using a selector list', () => {
```

Reply:
Yes of course. What I'm testing here is that the `delegateTarget` returns the element that was clicked on. I could have modified the existing test.